{"name": "test_fail7", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "def test_fail7():\n>       assert False\nE       assert False\n\ntest_fail.py:20: AssertionError"}, "start": 1747924138500, "stop": 1747924138500, "uuid": "85f32f90-9487-4a22-bf4c-73988817c663", "historyId": "e5476871c2923afbd50a8191a77cef77", "testCaseId": "e5476871c2923afbd50a8191a77cef77", "fullName": "tests.simple.test_fail#test_fail7", "labels": [{"name": "parentSuite", "value": "tests.simple"}, {"name": "suite", "value": "test_fail"}, {"name": "host", "value": "MacBook-Pro-Nikita.local"}, {"name": "thread", "value": "19780-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.simple.test_fail"}]}